src/Tizen.NUI.Components/Controls/RecyclerView/Item/RecyclerViewItem.cs

```diff
@@ -153,7 +153,7 @@ public bool IsSelected
         /// <summary>
         /// Flag to decide enable or disable in RecyclerViewItem.
```

Comment:
I don't understand, please change the description. "decide whether enable or disable ==what=="